README.md

```diff
@@ -43,6 +43,39 @@ Zapoznaj się z [instrukcją](CONTRIBUTING.md) zgłaszania Pull Requestów.
 * [Kurs HTML](http://www.kurshtml.edu.pl) - Rozwinięty kurs front-endowy.
 * [How2HTML](http://how2html.pl/) - Kurs HTML dla początkujących.
 * [CSS-Tricks](https://css-tricks.com/) - Blog na którym można znaleźć wiele przydatnych informacji na temat CSS.
+* [The awwwesomes](https://the-awwwesomes.gitbooks.io/html-css-step-by-step/content/pl/) - HTML i CSS wyjaśnione w przyjazny sposób i całkowicie od podstaw.
+* [Moja pierwsza strona internetowa w HTML5 i CSS](http://ferrante.pl/books/html/) - Od zera HTML i CSS.
+* [You Don't Know JS](https://github.com/getify/You-Dont-Know-JS) [en] - Seria darmowych ebooków na temat JS, spokojnie można też znaleźć na internecie książki do kupienia w języku polskim.
+* [Kurs React.js](https://typeofweb.com/kurs/react-js/) - Kurs Reacta od podstaw.
+* [Web Developer Roadmap 2018](https://github.com/kamranahmedse/developer-roadmap)
+* [awesome-react](https://github.com/enaqx/awesome-react) - Repo z linkami związanymi z Reactem.
+* [Flexbox Froggy](http://flexboxfroggy.com/) - Gra z żabkami :D dla początkujących, pomagająca zrozumieć działanie flexboxa.
+* [kurs.js](http://kursjs.pl/) - Kurs JavaScript od podstaw.
+* [codewars](https://www.codewars.com/) [en] - zadania do rozwiązywania online, różne języki (nie tylko te związane z webem)
+* [Sass cheatsheet](https://devhints.io/sass)
+* [Git Tutorial](https://try.github.io/levels/1/challenges/1) - Tutorial do nauki obsługi gita z linii komend.
+
+#### Blogi
+
+* [Na Frontendzie](https://www.nafrontendzie.pl/) - Czyli tak, jak mówi nazwa bloga.
+* [Kot źródłowy](http://kot-zrodlowy.pl/) - JS, chociaż i ogólnie to, co kręci się wokół web developmentu.
+* [Poradnik Javascript](http://poradnik.drogimex.pl/) - Fajne artykuły głównie na temat JavaScriptu.
+* [JS.n00b](http://jsdn.pl/) - JavaScript i m.in.: testowanie kodu, TypeScript, MongoDB.
+* [Semantyczny blog w HTML5](https://tutorials.comandeer.pl/html5-blog.html) - Dobrze wyjaśniony HTML 5 na przykładzie bloga.
```

Comment:
To raczej powinno znaleźć się w sekcji "Wybrane artykuły".

Reply:
hmm dlaczego?

Czy mówimy tylko o Twoim "semantycznym blogu" czy też o tych 3 powyżej?

Reply:
Tylko o moim. Bo to artykuł jest, nigdy większej ambicji w stosunku do tego nie miałem ;)

Reply:
Okej to racja 👍